src/player.h

```diff
@@ -567,7 +571,7 @@ class player : public Character, public JsonSerializer, public JsonDeserializer
         /** Returns true if the player scores a critical hit */
         bool scored_crit( float target_dodge = 0.0f ) const;
         /** Returns cost (in moves) of attacking with given item (no modifiers, like stuck) */
```

Comment:
Minor: could update the function description now we no longer have stuck weapons